realtime/server.js

```diff
@@ -21,14 +21,14 @@ var people = {};
 var usernames = {};
 var messages_typing = {};
 
-function getOptions(form, target, method) {
+function getOptions(form, params, method) {
```

Comment:
I think `target` is not  ab etter name because it's usually refering to a target for the message like "<channel_name>" or "<username>"